pkg/network/ovn_kubernetes.go

```diff
@@ -901,14 +904,14 @@ func validateOVNKubernetes(conf *operv1.NetworkSpec) []error {
 		var err error
 		if oc.V4InternalSubnet != "" {
 			if !cnHasIPv4 {
-				out = append(out, errors.Errorf("v4InternalSubnet and ClusterNetwork must have matching IP families"))
+				out = append(out, errors.Errorf("v4InternalSubnet %v and ClusterNetwork must have matching IP families"))
 			}
 			_, v4Net, err = net.ParseCIDR(oc.V4InternalSubnet)
 			if err != nil {
 				out = append(out, errors.Errorf("v4InternalSubnet is invalid: %s", err))
 			}
 			if !isV4InternalSubnetLargeEnough(conf) {
-				out = append(out, errors.Errorf("v4InternalSubnet is no large enough for the maximum number of nodes which can be supported by ClusterNetwork"))
+				out = append(out, errors.Errorf("v4InternalSubnet is not large enough for the maximum number of nodes which can be supported by ClusterNetwork"))
```

Comment:
let's print what v4InternalSubnet is

Reply:
this is the join subnet which is not part of this feature, but we can go ahead and update that log